GitUI/UserControls/RevisionGrid/RevisionGridControl.cs

```diff
@@ -131,7 +131,20 @@ public RevisionGridControl()
             InitializeComponent();
             InitializeComplete();
 
-            _loadingImage = new LoadingControl();
+            _loadingControl = new Label
+            {
+                Location = new Point(0, 0),
+                Dock = DockStyle.Right,
+                Padding = new Padding(7, 5, 5, 5),
```

Comment:
I suspect this needs to be scaled with `DipUtil`.

Reply:
I copied this from QuichSearchProvider.cs (line 30-40). I attempted replace this with a transparent spinner in the top-right corner, but failed to get it transparent. I do plan to make the loading indicator better. Will do it in a seperate pull request if that's ok with you. I also would like a more present loading indicator for the period that no row is returned from git.exe yet.